tests/pkg-relative-url/foo/BAR/baz.opf

```diff
@@ -14,8 +14,7 @@
     <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
```

Comment:
Same for the description on line 8 here, and page 1.

Reply:
See https://github.com/w3c/epub-tests/pull/96#discussion_r762531400